third_party/move/move-bytecode-verifier/transactional-tests/tests/type_safety/type_error_after_branch.v2_exp

```diff
@@ -0,0 +1,10 @@
+processed 1 task
```

Comment:
This file is consistent with third_party/move/move-bytecode-verifier/transactional-tests/tests/type_safety/type_error_after_branch.exp